docker/worker/worker.py

```diff
@@ -557,7 +557,7 @@ def _do_process_task(self, subscriber, subscription, ack_id, message,
         _state.source_id = None
         subscriber.acknowledge(subscription=subscription, ack_ids=[ack_id])
     except Exception:
-      logging.error('Unexpected exception while processing task: %s',
+      logging.error('Unexpected exception while processing task:\n%s',
```

Comment:
Maybe there's something we can do here where: if we're ever in an exception handling context, automatically attach the exception traceback after a newline.

Reply:
oh wait, I think this is exactly what `logging.exception` is for: https://docs.python.org/3/library/logging.html#logging.Logger.exception

Reply:
can we change up all of these cases to do that instead?